packages/core/src/baseclient.ts

```diff
@@ -258,6 +258,15 @@ export abstract class BaseClient<O extends ClientOptions> implements Client<O> {
     }
   }
 
+  /**
+   * Gets an installed integration by its `id`.
+   *
+   * @returns The installed integration or `undefined` if no integration with that `id` was installed.
+   */
+  public getIntegrationById(integrationId: string): Integration | undefined {
```

Comment:
What do you think about making this should be generic over an Integration? This means the caller can define the value without needing a cast.

```suggestion
  public getIntegrationById<I extends Integration>(integrationId: string): I | undefined {
```